### ISSUE TITLE
Staging Review finding: use va-breadcrumbs component for navigation on details pages for upcoming, pending, and past appointments

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#88443]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Appointments
**Product:** Appointments
**Feature:** Appointment Details Redesign

## Findings details
**VA.gov Experience Standard - issue:** [User encounters a component or pattern that isn't considered a best practice.](https://design.va.gov/about/experience-standards/#user-encounters-components-and-patterns-that-meet-best-practices)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @it-harrison (QA)

### Description

The navigation on the details pages is not rendered with the va-breadcrumbs component

## Recommended action

Use the va-breadcrumbs component

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** [Breadcrumbs ](https://design.va.gov/components/breadcrumbs)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [x] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [x] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [x] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by outerpress** [comment]
@it-harrison [here's Slack the conversation](https://dsva.slack.com/archives/C01DBGX4P45/p1721143680620019?thread_ts=1719870042.727779&cid=C01DBGX4P45) I mentioned. It sounds like @humancompanion-usds and @sterkenburgsara are leaning toward links for this component.

Since it's non-trivial to update these (we also use back link in the scheduling flow), we're going to wait until the pattern is defined in the design system. Is there a DS ticket we could add as a blocker to this? 

FYI @ldelacosta

**Comment by shiragoodman** [comment]
@outerpress thanks for sharing your plan, however we ask that tickets remain open until they're resolved. Please lmk if you'd like to discuss this further.

**Comment by outerpress** [comment]
Gotcha, thanks @shiragoodman - is there a ticket we can use to track the work for adding the back link pattern to the DS?

**Comment by shiragoodman** [comment]
@outerpress if there is, it would be in the [vets-design-system-documentation repo. ](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues) Tagging @caw310 as she may be able to help point you to it.

**Comment by outerpress** [comment]
I found 3 that seem related - @caw310 let me know if there are any others out there:

[Experimental Design Back Link #2797](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2797)

[Improvement to breadcrumb guidance](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2950)

[Update breadcrumb spacing in Page templates - Design](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3049)

Also fyi @kfink24 @Cieramaddox in case y'all are looking at spacing for the breadcrumbs/back links

**Comment by outerpress** [comment]
FYI @ldelacosta we'll want to wait for the following work to be done before moving forward:

> - Rachel Shearer has done some great work in Form step templates in Figma. Those will be coming in the next few weeks in the Design System pattern templates.
> - Robert Hasselle is also actively creating new mock-form pattern examples to use the minimal header, back link, etc.
> - The forms team is also building the 21-4138 form with minimal header, back link, etc. (I'm christening this as: "Form step layout v2" so we can talk about it more easily).
> - DST will use the Experimental Design Back Link issue you created to handle the component work.

This will also unblock the tickets to remove the back button from the scheduling flows.

See this thread for more info:
https://dsva.slack.com/archives/C01DBGX4P45/p1722016500668199?thread_ts=1719870042.727779&cid=C01DBGX4P45

**Comment by outerpress** [comment]
We'll handle this work in #93012

**Comment by outerpress** [comment]
Dev work is done, closing this cc ticket @JRRoof @Bren22va
